### PR TITLE
Remove redundant include in pinmux.c

### PR DESCRIPTION
**Pull request by alphan**
This line was added in #2031.

### CODE REVIEW

File: sw/device/lib/pinmux.c [hunk 1]
```diff
@@ -4,11 +4,9 @@
 
 #include "sw/device/lib/pinmux.h"
 
-#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
-#include "sw/device/lib/common.h"
-
 #include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
 #include "pinmux_regs.h"  // Generated.
+#include "sw/device/lib/common.h"
```
Comment (lenary) [inline sw/device/lib/pinmux.c:9]: Nit: Can we try to keep the generated and ungenerated headers separate? I might look at whether I can do this with clang-format going forwards.

Comment (gkelly) [reply sw/device/lib/pinmux.c:9]: +1

Comment (alphan) [reply sw/device/lib/pinmux.c:9]: Unfortunately, that wasn't me, that was `clang-format`. I can do the change without running `clang-format` but I'm not sure if we will have any issues in CI. (The version that I have is different and I didn't have a chance to play with docker yet).

Comment (lenary) [reply sw/device/lib/pinmux.c:8]: Ok, no worries, I'll add this to my list to look at today.

Comment (alphan) [reply sw/device/lib/pinmux.c:8]: How about now?

Comment (alphan) [reply sw/device/lib/pinmux.c:8]: Updated the commit, does it look better now?

**Pull request merged**